<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckBox</class>
 <widget class="QWidget" name="CheckBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Choosen Option</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkOption01">
        <property name="text">
         <string>Option 01</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkOption02">
        <property name="text">
         <string>Option 02</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkOption03">
        <property name="text">
         <string>Option 03</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
